avalon/pipeline.py

```diff
@@ -1901,9 +1901,15 @@ def path_from_represenation():
             return path
 
         normalized_path = os.path.normpath(path)
+        # This is fixing other bug down the integrator where
+        # normpath is used on template, messing with slashes.
+        # Because those are already saved in database, this is the
+        # place to fix it.
+        if platform.system().lower() != "windows":
+            normalized_path = normalized_path.replace("\\", "/")
         if os.path.exists(normalized_path):
             return normalized_path
-        return path
+        return normalized_path
```

Comment:
I think this returned "not normalized path" because normalized path may hide where the issue was...

Reply:
Ok, disregard then. But I believe that if the last test is made against `normalized_path` then it should return it.

Reply:
I think that's only reason why there is
```
if os.path.exists(normalized_path):
    # Everythink is ok then return normalized path
    return normalized_path
# Path is not right (maybe had `\\` in path which would be by nomalization converted to `\` == HARD TO DEBUG)
return path

```